packages/draft-js-export-html/src/stateToHTML.js

```diff
@@ -30,7 +30,7 @@ type RenderConfig = {
   style?: StyleDescr;
 };
 
-type BlockRenderer = (block: ContentBlock) => ?string;
+type BlockRenderer = (block: ContentBlock, generator?: MarkupGenerator) => ?string;
```

Comment:
I think you will also need to define a type for `MarkupGenerator` in this file (not to be confused with the typescript definitions which are totally different).